Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.py

```diff
@@ -110,7 +138,7 @@ def get_indicators(client: Client, args: Dict[str, Any]) -> CommandResults:
 
     results = CommandResults(
         outputs_prefix='Dragos.Indicators',
-        outputs_key_field='indicator_id',
+        outputs_key_field='id',
```

Comment:
does the old integration version return `indicator_id` in the context output and from now it will return only `id` ? if so this are breaking change and we should know about this